react/components/ListsPage/index.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { Component, Fragment, ReactNode } from 'react'
+import { withRuntimeContext } from 'vtex.render-runtime'
+import { Spinner } from 'vtex.styleguide'
+
+import { concat, filter, findIndex, map, update } from 'ramda'
+import { compose, withApollo, WithApolloClient } from 'react-apollo'
+import { InjectedIntlProps, injectIntl } from 'react-intl'
+
+import {
+  createList,
+  getListsFromLocaleStorage,
+  saveListIdInLocalStorage,
+} from '../../GraphqlClient'
+
+import Content from './Content'
+import ListSelector from './ListSelector'
+
+import wishlist from '../../wishList.css'
+
+const ON_LISTS_PAGE_CLASS = 'vtex-lists-page'
+
+interface ListsPageState {
+  lists?: any
+  selectedListId?: string
+  isLoading?: boolean
+}
+
+interface ListsPageProps extends InjectedIntlProps, WithApolloClient<any> {
+  runtime: any
+}
+
+class ListsPage extends Component<ListsPageProps, ListsPageState> {
+  public state: ListsPageState = {
```

Comment:
do you need to explicitly type the `state`?

Reply:
No, but isn't it a good practice?

Reply:
I think ts infers this automatically